sales_team_security_crm/security/security.xml

```diff
@@ -1,40 +1,35 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<odoo>
-    <data noupdate="1">
```

Comment:
I have run in local pre-commit, and this is not being modified by it, so better to keep the same 2 level indentation for not having too much noise in the diff.